SharpinitUnitTest/UnitRegistryTests.cs

```diff
@@ -0,0 +1,222 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using SharpInit.Platform;
+using SharpInit.Units;
+using System;
+
+namespace UnitTests.Units
+{
+    [TestClass]
+    public class UnitRegistryTests
+    {
+
+        [ClassInitialize]
+        public static void TestInitialize(TestContext value)
+        {
+            PlatformUtilities.RegisterImplementations();
+            PlatformUtilities.GetImplementation<IPlatformInitialization>().Initialize();
+            UnitRegistry.InitializeTypes();
+            UnitRegistry.ScanDefaultDirectories();
+        }
+
+        [TestCleanup]
+        public void TestCleanup()
+        {
+            UnitRegistry.Units.Clear();
+        }
+
+
+        [TestMethod]
+        public void AddUnit_UnitIsAdded_True()
+        {
+            // Arrange
+            Unit unit = new ServiceUnit("C:\\Users\\DJ\\source\\repos\\sharpsuite\\sharpinit\\SharpInit\\Units\\unit-test-files\\sshd.service");
```

Comment:
These are specific to your computer only, perhaps testing needs a Unit constructor with the contents of the unit file (so you can specify them right here)